9/11653.py

```diff
@@ -0,0 +1,10 @@
+n = int(input())
+
+if n == 1:
+    print('')
+
+for i in range(2, n+1):
+    if n % i == 0:
+        while n % i == 0: # 계속 나눠 줌
```

Comment:
거의 모든 프로그래밍에서 0은 False, 이외의 수는 True를 의미해요
```suggestion
        while not n % i: # 계속 나눠 줌
```
로 바꾸어 쓸 수 있겠네요